website/docs/r/spectrum_app.html.markdown

```diff
@@ -0,0 +1,54 @@
+---
+layout: "cloudflare"
+page_title: "Cloudflare: cloudflare_spectrum_app"
+sidebar_current: "docs-cloudflare-resource-load-balancer"
+description: |-
+  Provides a Cloudflare Sprectrum Application resource.
+---
+
+# cloudflare_spectrum_app
+
+Provides a Cloudflare Load Balancer resource.  You can extend the power of Cloudflare's DDoS, TLS, and IP Firewall to your other TCP-based services. This allows you to proxy tcp traffic over the Cloudflare network.
```

Comment:
```suggestion
Provides a Cloudflare Load Balancer resource. You can extend the power of Cloudflare's DDoS, TLS, and IP Firewall to your other TCP-based services. 
```

Reply:
Actually, Spectrum works with UDP too (internal beta for now).